bin/compile

```diff
@@ -42,6 +42,7 @@ if [ -n "$LANG" ]; then
   puts_warn "You should switch to a '.locales' file."
   puts_warn "See https://github.com/heroku/heroku-buildpack-locale"
 
+  test `tail -c 1 $BUILD_DIR/.locales` && echo "\n" >> $BUILD_DIR/.locales
```

Comment:
`echo` already produces a line break so `\n` is going to add two. You want `echo ""` or `echo -n "\n"`.

Reply:
:+1: